sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/FetchUpdatedDataSource.groovy

```diff
@@ -0,0 +1,43 @@
+package life.qbic.business.notification.create
+
+import life.qbic.business.exception.DatabaseQueryException
+import life.qbic.datamodel.samples.Status
+import life.qbic.business.subscription.Subscriber
+
+import java.time.LocalDate
+
+/**
+ * <p>Fetches the subscribers from the database</p>
+ *
+ * @since 1.0.0
+ */
+interface FetchUpdatedDataSource {
```

Comment:
What is the idea of this interface? Why did you add it? Was there a method missing in the old interface? Or in which ways does it differ to the other DS?